lib/widgets-page.php

```diff
@@ -36,8 +36,8 @@ class="blocks-widgets-container
  * @param string $hook Page.
  */
 function gutenberg_widgets_init( $hook ) {
-	if ( 'gutenberg_page_gutenberg-widgets' !== $hook && 'gutenberg_customizer' !== $hook ) {
-			return;
+	if ( ! in_array( $hook, array( 'gutenberg_page_gutenberg-widgets', 'gutenberg_customizer', 'widgets.php' ) ) ) {
```

Comment:
We don't need to run _all_ of this function in `widgets.php`, just `wp_enqueue_style( 'wp-block-library' );`